Code/MinimalLib/jswrapper.cpp

```diff
@@ -564,7 +575,14 @@ EMSCRIPTEN_BINDINGS(RDKit_minimal) {
       .function("get_num_atoms",
                 select_overload<unsigned int() const>(&JSMol::get_num_atoms))
       .function("get_num_bonds", &JSMol::get_num_bonds)
-  ;
+#ifdef RDK_BUILD_MINIMAL_LIB_MMPA
```

Comment:
I think `allow_raw_pointers()` is not necessary, since we are returning a JavaScript object containing `MolList` object. The user needs to take care deleting the inner `MolList` objects anyway (as with all C++ objects), but JS should take care of deleting the outer JS wrapper object. I cannot seem to be able to find a similar example on SO, but this compiles OK and works at runtime, so it should be good without `allow_raw_pointers()`.

Reply:
@ptosco should it be changed, or is it ok as is?